content/contributing/project-information/bevy-organization.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "The Bevy Organization"
+insert_anchor_links = "right"
+[extra]
+weight = 2
++++
+
+The Bevy Organization is the group of people responsible for stewarding the Bevy project. It handles things like merging pull requests, choosing project direction, managing bugs / issues / feature requests, running the Bevy website, controlling access to secrets, defining and enforcing best practices, etc.
```

Comment:
```suggestion
The Bevy Organization is the group of people responsible for stewarding the Bevy Project. It handles things like merging pull requests, choosing project direction, managing bugs / issues / feature requests, running the Bevy website, controlling access to secrets, defining and enforcing best practices, etc.
```
This is capitalized as a proper noun in the `_index.md` of this section, so I'm presuming this should be capitalized.